on OSX, tests fail

Tests fail on OSX when they are enabled ( -DBUILD_ZETAGLEST_TESTS=ON)

https://travis-ci.org/ZetaGlest/zetaglest-source/jobs/429367334#L2012

```
Scanning dependencies of target zetaglest_tests
[ 97%]Building CXX object source/tests/CMakeFiles/zetaglest_tests.dir/test_runner.cpp.o
[ 97%]Building CXX object source/tests/CMakeFiles/zetaglest_tests.dir/shared_lib/graphics/font_test.cpp.o
In file included from /Users/travis/build/ZetaGlest/zetaglest-source/source/tests/shared_lib/graphics/font_test.cpp:12:
In file included from /usr/local/include/cppunit/extensions/HelperMacros.h:9:
/usr/local/include/cppunit/TestCaller.h:121:28:error:no member named 'bind' in namespace 'std'; did you mean 'find'?
            m_test_function( std::bind(test, m_fixture) )
                             ~~~~~^
/Applications/Xcode-9.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/algorithm:888:1:note:'find' declared here
find(_InputIterator __first, _InputIterator __last, const _Tp& __value_)
^
In file included from /Users/travis/build/ZetaGlest/zetaglest-source/source/tests/shared_lib/graphics/font_test.cpp:12:
In file included from /usr/local/include/cppunit/extensions/HelperMacros.h:9:
/usr/local/include/cppunit/TestCaller.h:138:28:error:no member named 'bind' in namespace 'std'; did you mean 'find'?
            m_test_function( std::bind(test, &fixture) )
                             ~~~~~^
/Applications/Xcode-9.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/algorithm:888:1:note:'find' declared here
find(_InputIterator __first, _InputIterator __last, const _Tp& __value_)
^
In file included from /Users/travis/build/ZetaGlest/zetaglest-source/source/tests/shared_lib/graphics/font_test.cpp:12:
In file included from /usr/local/include/cppunit/extensions/HelperMacros.h:9:
/usr/local/include/cppunit/TestCaller.h:155:28:error:no member named 'bind' in namespace 'std'; did you mean 'find'?
            m_test_function( std::bind(test, fixture) )
                             ~~~~~^
/Applications/Xcode-9.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/algorithm:888:1:note:'find' declared here
find(_InputIterator __first, _InputIterator __last, const _Tp& __value_)
^
In file included from /Users/travis/build/ZetaGlest/zetaglest-source/source/tests/shared_lib/graphics/font_test.cpp:12:
In file included from /usr/local/include/cppunit/extensions/HelperMacros.h:9:
/usr/local/include/cppunit/TestCaller.h:155:23:error:no matching function for call to 'find'
            m_test_function( std::bind(test, fixture) )
                             ^~~~~~~~~
/Users/travis/build/ZetaGlest/zetaglest-source/source/tests/shared_lib/graphics/font_test.cpp:33:2:note:in instantiation of member function 'CppUnit::TestCaller<FontTest>::TestCaller' requested here
        CPPUNIT_TEST( test_LTR_RTL_Mixed );
        ^
/usr/local/include/cppunit/extensions/HelperMacros.h:299:15:note:expanded from macro 'CPPUNIT_TEST'
        ( new CPPUNIT_NS::TestCaller<TestFixtureType>(    \
              ^
/usr/local/include/cppunit/Portability.h:107:21:note:expanded from macro 'CPPUNIT_NS'
# define CPPUNIT_NS CppUnit
                    ^
/Applications/Xcode-9.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/algorithm:888:1:note:candidate function template not viable: requires 3 arguments, but 2 were provided
find(_InputIterator __first, _InputIterator __last, const _Tp& __value_)
^
4 errors generated.
make[2]: *** [source/tests/CMakeFiles/zetaglest_tests.dir/shared_lib/graphics/font_test.cpp.o] Error 1
make[1]: *** [source/tests/CMakeFiles/zetaglest_tests.dir/all] Error 2
make: *** [all] Error 2
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -62,7 +62,7 @@ script:
   - if [ "$TRAVIS_OS_NAME" = "linux" ] && [ "$(echo "$CC" | grep 'clang')" = "" ]; then
         mkdir build;
         cd build;
-        cmake ..  -DCMAKE_INSTALL_PREFIX=$PWD/local -DBUILD_MODEL_VIEWER=ON -DBUILD_MAP_EDITOR=ON -DBUILD_IMPORT_EXPORT_TOOLS=ON -DBUILD_ZETAGLEST_TESTS=ON;
+        cmake ..  -DCMAKE_INSTALL_PREFIX=$PWD/local -DBUILD_MODEL_VIEWER=ON -DBUILD_MAP_EDITOR=ON -DBUILD_MODEL_IMPORT_EXPORT_TOOLS=ON -DBUILD_ZETAGLEST_TESTS=ON;
         make && make install;
     fi
   - if [ "$TRAVIS_OS_NAME" = "linux" ] && [ "$(echo "$CC" | grep 'clang')" != "" ]; then
@@ -74,8 +74,7 @@ script:
   - if [ "$TRAVIS_OS_NAME" = "osx" ]; then
         mkdir build;
         cd build;
-        export CXXFLAGS="-std=c++11";
-        cmake .. -DCMAKE_INSTALL_PREFIX=$PWD/local -DBUILD_MODEL_VIEWER=ON -DBUILD_MAP_EDITOR=ON -DBUILD_IMPORT_EXPORT_TOOLS=ON -DBUILD_ZETAGLEST_TESTS=ON;
+        cmake .. -DCMAKE_INSTALL_PREFIX=$PWD/local -DBUILD_MODEL_VIEWER=ON -DBUILD_MAP_EDITOR=ON -DBUILD_MODEL_IMPORT_EXPORT_TOOLS=ON -DBUILD_ZETAGLEST_TESTS=ON;
         make && make install;
         echo 'Dependencies:';
         otool -L zetaglest;

--- a/.travis.yml
+++ b/.travis.yml
@@ -77,9 +77,6 @@ script:
         export CXXFLAGS="-std=c++11";
         cmake .. -DCMAKE_INSTALL_PREFIX=$PWD/local -DBUILD_MODEL_VIEWER=ON -DBUILD_MAP_EDITOR=ON -DBUILD_IMPORT_EXPORT_TOOLS=ON -DBUILD_ZETAGLEST_TESTS=ON;
         make && make install;
-       # echo 'Mini test:';
-       # echo '>>> zetaglest --version';
-       # ./zetaglest --version | head -3;
         echo 'Dependencies:';
         otool -L zetaglest;
     fi

--- a/.travis.yml
+++ b/.travis.yml
@@ -50,7 +50,9 @@ before_install:
     # ^ odd linking problems related with brew which is not creating links
   - if [ "$TRAVIS_OS_NAME" = "osx" ] && [ "$Tr_Xcode_Version" = "6.4" ]; then brew install Caskroom/cask/xquartz; fi
   - if [ "$Tr_Compiler_Version" != "" ] && [ "$Tr_Compiler_Version" != "default" ] && [ "$CC" = "gcc" ]; then export CXX="g++-${Tr_Compiler_Version}" CC="gcc-${Tr_Compiler_Version}"; fi
-  - if [ "$Tr_Compiler_Version" != "" ] && [ "$Tr_Compiler_Version" != "default" ] && [ "$CC" == "clang" ]; then export CXX="clang++-${Tr_Compiler_Version}" CC="clang-${Tr_Compiler_Version}"; fi
+  - if [ "$Tr_Compiler_Version" != "" ] && [ "$Tr_Compiler_Version" != "default" ] && [ "$CC" == "clang" ]; then
+      export CXX="clang++-${Tr_Compiler_Version}" CC="clang-${Tr_Compiler_Version}";
+    fi
   - $CC --version
   - $CXX --version
   - cmake --version
@@ -72,11 +74,12 @@ script:
   - if [ "$TRAVIS_OS_NAME" = "osx" ]; then
         mkdir build;
         cd build;
-        cmake .. -DCMAKE_INSTALL_PREFIX=$PWD/local -DBUILD_MODEL_VIEWER=ON -DBUILD_MAP_EDITOR=ON -DBUILD_IMPORT_EXPORT_TOOLS=ON;
+        export CXXFLAGS="-std=c++11";
+        cmake .. -DCMAKE_INSTALL_PREFIX=$PWD/local -DBUILD_MODEL_VIEWER=ON -DBUILD_MAP_EDITOR=ON -DBUILD_IMPORT_EXPORT_TOOLS=ON -DBUILD_ZETAGLEST_TESTS=ON;
         make && make install;
-        echo 'Mini test:';
-        echo '>>> zetaglest --version';
-        ./zetaglest --version | head -3;
+       # echo 'Mini test:';
+       # echo '>>> zetaglest --version';
+       # ./zetaglest --version | head -3;
         echo 'Dependencies:';
         otool -L zetaglest;
     fi

--- a/BUILD.md
+++ b/BUILD.md
@@ -72,7 +72,7 @@ To edit maps:
 
 Use model import export tools:
 
-    BUILD_IMPORT_EXPORT_TOOLS=ON (off by default)
+    BUILD_MODEL_IMPORT_EXPORT_TOOLS=ON (off by default)
 
 The **wxWidgets (libwxgtk3.0-dev on some systems)** and **libxml2-dev*
 dependencies (mentioned below) is only required if you enable those

--- a/source/tests/CMakeLists.txt
+++ b/source/tests/CMakeLists.txt
@@ -7,6 +7,11 @@ SET(TARGET_NAME "zetaglest_tests")
 IF(BUILD_ZETAGLEST_TESTS)
 	MESSAGE(STATUS "Build ${TARGET_NAME} = YES")
 
+# The tests fail on Travis with clang on osx because of the use of "bind"
+	if("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
+		SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
+	endif()
+
 	INCLUDE (CheckIncludeFiles)
 
 	FIND_PACKAGE(CppUnit REQUIRED)

